smart-contract-verifier/smart-contract-verifier-server/src/types/vyper_standard_json.rs

```diff
@@ -1,166 +0,0 @@
-
-impl From<VerifyVyperStandardJsonRequest> for VerifyVyperStandardJsonRequestWrapper {
-    fn from(inner: VerifyVyperStandardJsonRequest) -> Self {
-        Self(inner)
-    }
-}
-
-impl Deref for VerifyVyperStandardJsonRequestWrapper {
-    type Target = VerifyVyperStandardJsonRequest;
-
-    fn deref(&self) -> &Self::Target {
-        &self.0
-    }
-}
-
-impl VerifyVyperStandardJsonRequestWrapper {
-    pub fn new(inner: VerifyVyperStandardJsonRequest) -> Self {
-        Self(inner)
-    }
-
-    pub fn into_inner(self) -> VerifyVyperStandardJsonRequest {
-        self.0
-    }
-}
-
-impl TryFrom<VerifyVyperStandardJsonRequestWrapper> for VerificationRequest {
-    type Error = StandardJsonParseError;
-
-    fn try_from(request: VerifyVyperStandardJsonRequestWrapper) -> Result<Self, Self::Error> {
-        let request = request.into_inner();
-
-        let bytecode = DisplayBytes::from_str(&request.bytecode)
-            .map_err(|err| anyhow!("Invalid deployed bytecode: {:?}", err))?
-            .0;
-        let (creation_bytecode, deployed_bytecode) = match request.bytecode_type() {
-            BytecodeType::Unspecified => Err(StandardJsonParseError::BadRequest(anyhow!(
-                "Bytecode type is unspecified"
-            )))?,
-            BytecodeType::CreationInput => (Some(bytecode), bytes::Bytes::new()),
-            BytecodeType::DeployedBytecode => (None, bytecode),
-        };
-        let compiler_version = DetailedVersion::from_str(&request.compiler_version)
-            .map_err(|err| anyhow!("Invalid compiler version: {}", err))?;
-
-        let input: CompilerInput = serde_json::from_str(&request.input)?;
-
-        Ok(Self {
-            deployed_bytecode,
-            creation_bytecode,
-            compiler_version,
-            content: StandardJsonContent { input },
-            chain_id: request.metadata.and_then(|metadata| metadata.chain_id),
-        })
-    }
-}
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-    use crate::proto::VerificationMetadata;
-    use pretty_assertions::assert_eq;
-
-    #[test]
-    fn try_into_verification_request() {
-        /********** Creation Input **********/
-
-        let mut request = VerifyVyperStandardJsonRequest {
-            bytecode: "0x1234".to_string(),
-            bytecode_type: BytecodeType::CreationInput.into(),
-            compiler_version: "v0.3.7+commit.6020b8bb".to_string(),
-            input: "{\"language\":\"Vyper\",\"sources\":{\"Contract.vy\":{\"content\":\"stored: address\\r\\n\\r\\nevent SingleVyper17: pass\\r\\n\\r\\n@internal\\r\\ndef extract(temp: address) -> address:\\r\\n  ret: address = self.stored\\r\\n  self.stored = temp\\r\\n  return ret\\r\\n\\r\\n@external\\r\\ndef identity(x: address) -> address:\\r\\n  log SingleVyper17()\\r\\n  temp: address = self.stored\\r\\n  self.stored = x\\r\\n  return self.extract(temp)\"}},\"settings\":{\"outputSelection\":{\"*\":[\"evm\"]}}}".to_string(),
```

Comment:
You mentioned that these tests will be brought back in the next pr's, right? Just to clarify

Reply:
Yeap, it has been returned back in https://github.com/blockscout/blockscout-rs/pull/1308